applications/TrilinosApplication/trilinos_space.h

```diff
@@ -1004,6 +1016,91 @@ class TrilinosSpace
         KRATOS_CATCH("");
     }
 
+    /**
+     * @brief Generates a graph combining the graphs of two matrices
+     * @param rA The first matrix
+     * @param rB The second matrix
+     */
+    static Epetra_CrsGraph CombineMatricesGraphs(
```

Comment:
If you need to use this function from the outside (not only from within this file( would move this function to the trilinos utility class instead of the space.

If it is only an internal functionality, I would suggest making it private to make it clear that it is not part of the Space interface.

Reply:
I need it to be public in order to add a test